### PR TITLE
authentication and registration 

### PR DESCRIPTION
**Pull request by Amir-Gaifullin**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   resources :projects
   resources :tasks
+
+  get '/users', to: 'users#show'
```
Comment (DmitryBarskov) [inline config/routes.rb:5]: You don't need this line, since it is already stated in the next one.

File: .semaphore/heroku.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+version: v1.0
```
Comment (DmitryBarskov) [inline .semaphore/heroku.yml:1]: why do i see this file?

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+			render :new
+		end
+	end
+
+	private
+
+	def user_params
+		params.require(:user).permit(:email, :password)
+	end
+end
```
Comment (DmitryBarskov) [inline app/controllers/sessions_controller.rb:24]: don't use tabs anymore.

**Review by DmitryBarskov — CHANGES_REQUESTED** [review]
I don't see authorization here.

**Comment by Amir-Gaifullin** [comment]
@DmitryBarskov Now I have run this thread and I have authorization and registration here, do not understand the question?

**Comment by Amir-Gaifullin** [comment]
This pr for authentication and registration, autorization in "Autorization_Task"

**Review by DmitryBarskov — APPROVED** [review]
LGFM

**Pull request closed without merging**